pages/index.html

```diff
@@ -1,39 +1,167 @@
 ---
 permalink: /
-layout: styleguide
+layout: homepage
 title: U.S. Federal Web Design Standards
 ---
 
-<h1>About this Guide</h1> 
+<a class="skipnav" href="#main-content">Skip main navigation</a>
 
-<p>Welcome to the alpha site for the U.S. Federal Web Design Standards project.</p> 
+<header role="banner">
+  
+  <div class="usa-disclaimer">
+    <div class="usa-grid">
+      <span class="usa-disclaimer-official">
+        <img class="usa-flag_icon" alt="US flag signifying that this is a United States Federal Government website" src="{{ site.baseurl }}/assets/img/us_flag_small.png">
+        An official website of the United States Government
+      </span>
+      <span class="usa-disclaimer-stage">This site is currently in alpha. <a href="#">Learn more.</a></span>
+    </div>
+  </div>
+  
+  <section class="usa-banner">
+    <div class="usa-grid">
+      <nav>
+        <a class="usa-banner-link-top" href="https://github.com/18F/usfwds">View on GitHub</a>
+      </nav>
+      <div class="usa-banner-content" id="main-content">
+        <h1>U.S. Federal Web Design Standards</h1>
+        <h2 class="usa-lead">Open source UI components, visual style guide and content style guide for US federal government websites.</h2>
+      </div>
+        <a class="usa-button usa-button-big usa-button-secondary" href="{{ site.baseurl }}/getting-started">View the standards</a>
+        <div class="usa-button-block">
+          <a class="usa-button usa-button-big usa-button-outline-inverse">Download the components</a>
+          <p class="usa-text-small">Download a zip file with the assets of this library</p>
+        </div>
+    </div>
+  </section>
+  
+</header>
 
-<p>This web design standards is intended to provide reference for designers, developers, as well as anyone else interested in the design and frontend principles that make up the building blocks of federal government websites.</p>
+<section class="usa-section usa-section-alt">
+  <div class="usa-grid">
+    <div class="usa-intro">
+      <h2>Tools for creating beautiful online experiences for the American people.</h2>
+      <p class="intro-text">Built and maintained by a team of US Digital Service + 18F designers and developers, this resource is built on the highest standards of 508 compliance, reuses best practices of existing style libraries and modern web design to guide us in creating beautiful and easy-to-use online experiences for the American people.</p>
+    </div>
+    <div class="usa-width-one-half">
+      <a href="#">
+        <div class="usa-img-secondary">
+          <img src="{{ site.baseurl }}/assets/img/home/homepage_illustrations_ui_components.png" alt="UI Components">
+        </div>
+        <h3>UI Components</h3>
+      </a>
+      <p>Common web interactions (buttons, forms, navigation, etc.) with reusable and downloadable code</p>
+    </div>
+    <div class="usa-width-one-half usa-end-row">
```

Comment:
So what about this case:

In a year from now everybody gets a super large monitor and we decide to expand the whole pattern library and add another breakpoint for mega size screen. At this mega breakpoint we change 1/2 so it's actually 1/4 so we can get 4 by side. This won't work with a `usa-end-row` because we're forcing it to always break at 2 by 2. Theres also current cases where using an end row with 1/4 widths will break on smaller breakpoints, and that's happening in our current code.

So what I don't want to see: 1. a user looks at our code, see's where using this method for a grid with 1/2 items and then use the same method for a grid with 1/3 items and have it break on them. 2. creating code that can't adapt to changes in the future.